src/render.rs

```diff
@@ -144,6 +148,13 @@ gfx_defines! {
         color_target: gfx::RenderTarget<ColorFormat> = "Target0",
         depth_target: gfx::DepthTarget<DepthFormat> = gfx::preset::depth::LESS_EQUAL_WRITE,
     }
+
+    pipeline skybox_pipe {
```

Comment:
I think you can re-use `quad_pipe` here?

Reply:
I'll try